pkg/appgw/appgw_test.go

```diff
@@ -55,7 +56,8 @@ var _ = Describe("Tests `appgw.ConfigBuilder`", func() {
 			Name:      ingressName,
 			Namespace: ingressNS,
 			Annotations: map[string]string{
-				k8scontext.IngressClass: k8scontext.IngressControllerName,
+				annotations.IngressClassKey:   annotations.IngressControllerName,
+				annotations.BackendPathPrefix: "/",
```

Comment:
Please add test to verify/read the Annotations settings in - 
"Context("Tests Application Gateway Configuration", func()"